### PR TITLE
Present organizing committee

### PR DESCRIPTION
**Pull request by DonGiovanni83**
#22

### CODE REVIEW

File: layouts/_default/about.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{{ define "main" }}
+<div class="Container">
```
Comment (ioboi) [inline layouts/_default/about.html]: Please use only one container and wrap everything in it.

File: layouts/partials/about/member-role.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<ul>
+    {{ range .Params.roles }}
+    <li class="role">{{ . }}</li>
```
Comment (ioboi) [inline layouts/partials/about/member-role.html]: Look at the CSS Guidelines for naming.

The ul should have a class called RoleList and li should have a class called RoleList-Element.

Comment (DonGiovanni83) [reply layouts/partials/about/member-role.html]: should we even use a list for the roles? I think a subheader would work to

Comment (ioboi) [reply layouts/partials/about/member-role.html]: I think a list is the right choice. In the end we want to format this list as the badges were. I think from a CSS standpoint its much better to be explicit.

Comment (ioboi) [reply layouts/partials/about/member-role.html]: We must think about if we are using the semantic elements correctly, please check if we are using those correctly with this approach.

**Pull request merged**